modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalEditExternalReferenceCode.scss

```diff
@@ -0,0 +1,3 @@
+.lfr-object-web-modal-edit-external-reference-code {
+	overflow-y: hidden !important;
+}
```

Comment:
Hi @carolmariaabb. I don't know if you've tried to use a more specific class to avoid using !important. If onot, could you try to do this, please?

Reply:
I've tried but I couldn't fix the bug without using `!importat`.

The bug is that a scroll appears when the modal opens. But I will remove this class for now so that some FE can properly fix this issue in the future.

Thanks!

![image](https://user-images.githubusercontent.com/34186632/192275834-35b171ab-ee1d-4e7e-8c3f-4f09b3cdcf0c.png)